backend/news_api_scraper.py

```
@@ -0,0 +1,495 @@
+#!/usr/bin/env python3
+"""
+NewsAPI Scraper - FREE Industry News Monitoring
+Monitor news for contract announcements, industry trends, company updates
+Rate Limit: 1,000 requests/day (FREE)
+"""
+
+import os
+import sys
+import requests
+import sqlite3
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)